Libs/MRML/Widgets/qMRMLPlotView.cxx

```diff
@@ -604,6 +604,20 @@ void qMRMLPlotViewPrivate::RecalculateBounds()
   q->updateMRMLChartAxisRangeFromWidget();
 }
 
+// --------------------------------------------------------------------------
+void qMRMLPlotViewPrivate::ReleasePlotSelections()
```

Comment:
It could make sense to make it available as a public slot.
Also, we could find a more standard name, such as ```RemoveSelection```.

Reply:
ok